packages/backend-common/src/database/DatabaseManager.ts

```diff
@@ -47,19 +47,22 @@ export class DatabaseManager {
    * names if config is not provided.
    *
    * @param config - The loaded application configuration.
+   * @param runMigrations - Controls whether or not to perform database migrations.
    */
-  static fromConfig(config: Config): DatabaseManager {
+  static fromConfig(config: Config, runMigrations?: boolean): DatabaseManager {
```

Comment:
Let's throw this in an optional options object in stead so that you'd call it like so: `.fromConfig(config, { runMigrations: false})`